packages/common/src/types/ccdAmount.ts

```diff
@@ -22,6 +24,44 @@ export class CcdAmount {
         this.microCcdAmount = microCcdAmount;
     }
 
+    /**
+     * Returns the amount of CCD as a string.
+     *
+     * @param useCommaDecimalSeperator Whether or not to use comma as a decimal seperator, defaults to false
+     * @returns The amount of CCD as a string
+     */
+    toCcd(useCommaDecimalSeperator = false): string {
+        const ccd = Big(this.microCcdAmount.toString()).div(Big(1000000));
+
+        if (useCommaDecimalSeperator) {
+            return ccd.toString().replace('.', ',');
+        } else {
+            return ccd.toString();
+        }
+    }
+
```

Comment:
It would be good to add helper functions that are independent to the `CcdAmount` class such as

`function ccdToMicroCcd(string){ ... return string}`

`function microCcdToCcd(string){ ... return string}`

The above helper function can create an instance of the `CcdAmount` as part of its logic so (meaning it is hidden from the developer using the function that we need to deal with the `CcdAmount` class)

Reply:
I think this is more elegant, also not hard to implement.